src/main/java/teammates/ui/template/FeedbackResultsTable.java

```diff
@@ -24,7 +26,45 @@ public FeedbackResultsTable(int fbIndex, String studentName, FeedbackSessionResu
             for (Map.Entry<String, List<FeedbackResponseAttributes>> entry : received.entrySet()) {
                 giverIndex++;
                 this.receivedResponses.add(new FeedbackResponsePersonRow(fbIndex, giverIndex, entry.getKey(), "giver",
-                                                                         entry.getValue(), result));
+                        entry.getValue(), result));
+            }
+
+            boolean hasAnsweredContrib = false;
+            boolean contribExistsFromOtherStudent = false;
+            FeedbackResponseAttributes response = new FeedbackResponseAttributes();
+
+            for (Map.Entry<String, List<FeedbackResponseAttributes>> entry : received.entrySet()) {
+                for (int j = 0; j < entry.getValue().size(); j++) {
+                    FeedbackQuestionType questionType = entry.getValue().get(j).feedbackQuestionType;
+                    String recipient = entry.getValue().get(j).recipient;
+                    String giver = entry.getValue().get(j).giver;
+                    if (questionType.equals(FeedbackQuestionType.CONTRIB)) {
+                        response = new FeedbackResponseAttributes(entry.getValue().get(j));
+                        contribExistsFromOtherStudent = true;
+                    }
+                    if (giver.equals(recipient) && questionType.equals(FeedbackQuestionType.CONTRIB)) {
+                        hasAnsweredContrib = true;
+                    }
+                }
+            }
+            /*
+             * if student has not submitted the session or CONTRIB response
```

Comment:
Use `// ...` instead of `/* ... */` for block comments.

Reply:
sure.